packages/web3-core-method/src/index.js

```diff
@@ -56,6 +56,7 @@ var Method = function Method(options) {
     this.transactionBlockTimeout = options.transactionBlockTimeout || 50;
     this.transactionConfirmationBlocks = options.transactionConfirmationBlocks || 24;
     this.transactionPollingTimeout = options.transactionPollingTimeout || 750;
+    this.blockHeaderTimeout = options.blockHeaderTimeout || 10; // 10 seconds
```

Comment:
Any issue with cutting this down to 5 seconds?

Reply:
Personally I was not sure about this default value. I will update it to 5.